src/sortPackageJson.ts

```diff
@@ -1,9 +1,11 @@
-import { Range, TextEditor, TextEditorEdit, window } from 'vscode'
+import { Range, TextEditor, TextEditorEdit } from 'vscode'
 import { getSortFn } from './getSortFn'
 
 export function sortPackageJson(textEditor: TextEditor, edit: TextEditorEdit) {
 	if (!textEditor.document.fileName.endsWith('package.json')) {
-		window.showWarningMessage('Active file is not package.json')
+		console.info(
+			`editor.sortPackageJson is invoked but active file is ${textEditor.document.fileName} and not package.json. This is a concurrency bug from vscode.`
+		)
```

Comment:
@unional great, thanks for this!

For anyone wondering what changed here, here is a comparison between `unional.vscode-sort-package-json@1.5.3` and `unional.vscode-sort-package-json@1.5.4` check out this video:

https://github.com/user-attachments/assets/bcac2646-94a7-4204-bd73-278bce25e76f